File: Graph/GFG/Easy/GraphsWithNVertex/GraphsWithNVertex.cpp
// https://www.geeksforgeeks.org/problems/graph-and-vertices/1
#include<bits/stdc++.h>
using namespace std;
long long count(int n)
{
    long long ans = pow(2, (n * (n - 1) / 2));
    return ans;
}
int main(void) {
    cout << count(5);
    return 0;
}

File: Graph/GFG/Easy/GraphsWithNVertex/GraphsWithNVertex.java
// https://www.geeksforgeeks.org/problems/graph-and-vertices/1
public class GraphsWithNVertex {
    static long count(int n) {
        long ans = (long) Math.pow(2, n * (n - 1) / 2);
        return ans;
    }
    public static void main(String[] args) {
        System.out.println(count(5));
    }
}